thrift/events.thrift

```diff
@@ -1,3 +1,5 @@
+include "contentatom.thrift"
+
 typedef string ContentAtomID
 
 enum EventType { PUBLISH, UPDATE, TAKEDOWN }
```

Comment:
Yes, it does mean take an existing atom and make it live, but definitely this lifecycle needs to be specced out more thoroughly anyway.